src/main/scala/io/codeheroes/kafkalogger/HeroLogger.scala

```diff
@@ -0,0 +1,25 @@
+package io.codeheroes.kafkalogger
+
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.Appender
+import org.slf4j.Logger
+
+
+class HeroLogger(
```

Comment:
It would be greate to have companion object for this class with method single() which takes single appender - this will be helpful when someone wants to create logger with single appender :)